frontend/src/components/WineDescription.jsx

```diff
@@ -1,6 +1,25 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+
+// comment out the console log to get a glimpse of what's in there :)
+
+const WineDescription = (props) => {
+  const [myData, setMyData] = useState(null);
+  const wineId = props.match.params.wine_id;
+
+  // localhost will be changed, just here while in development
+  const getData = async () => {
+    const response = await axios.get(
+      `http://localhost:8080/api/wines/${wineId}`
+    );
+    setMyData(response.data);
+    // console.log(response.data);
+  };
+
+  useEffect(() => {
+    getData();
+  }, []);
```

Comment:
There's a potential bug here:
The `useEffect` has an empty list of  deps, meaning it only runs on mount. But if we ever update the `wineId`, the effect won't run again.
We should make the effect dependent on the `wineId`, also it's better to move the `getData` function to inside the effect.
https://overreacted.io/a-complete-guide-to-useeffect/#moving-functions-inside-effects